core/templates/dev/head/css/oppia.css

```diff
@@ -3111,3 +3112,275 @@ md-card.preview-conversation-skin-supplemental-card {
     float: right;
   }
 }
+
+.oppia-gallery-summary-tile {
```

Comment:
Hi @amgowano -- I believe we moved these to the exploration-summary-tile directive. It looks like you've deleted those and put them back here. Was that intentional? Might be an accidental artifact of the merge.